constants/constants.go

```diff
@@ -67,3 +67,16 @@ const (
 	// DockerHostMode is the string of the network mode that indicates a host namespace
 	DockerHostMode = "host"
 )
+
+type DockerMonitorMode int
+
+const (
+	// DockerMode is a mode for docker monitor when trireme is running on host with just a docker daemon
+	DockerMode DockerMonitorMode = iota
+
+	// KubernetesMode is a mode for docker monitor when trireme is running on host which is part of a kubernetes cluster
+	KubernetesMode
+
+	// ECSMode is a mode for docker monitor when trireme is running on host which is part of ECS/AWS cluster
```

Comment:
I think that from a trireme standpoint this is essentially a NoProxyMode and not ECS mode. The ECS aspects are covered on the metadata extractors. We should change the naming convention here. We should also provide a default metadata extractor for ECS. 

In the whole PR for Trireme, nothing is ECS specific. All references to ECS should be removed and changed to NoProxyMode (or something across these lines).

Reply:
Renamed ECS to NoProxyMode